core/spec/unit/rom/attribute/method_missing_spec.rb

```diff
@@ -3,7 +3,7 @@
 
 RSpec.describe ROM::Attribute, '#method_missing' do
   subject(:attribute) do
-    ROM::Attribute.new(type)
+    ROM::Attribute.new(type, name: :foo)
```

Comment:
There are lots of places in the test suite where `define_attribute` helper could be used. However, it is not flexible enough to accept some scenarios, like a type with call to `.optional`, or a type under `ROM::Memory` namespace. Maybe a nice refactor for consistency would be whether make it more flexible or get rid off it.

Reply:
I agree this could be improved, let's keep this in mind for future development. Feel free to add a `TODO` comment about it 😄